### DISCUSSION TITLE
Question:  Screen Area Sharing vs Window Sharing

### DISCUSSION THREAD

**Discussion by ccroussore** [MarcusWichelmann/MarcusW.VncClient#15] [Q&A] [answered]
We are looking for a library that will allow us to share a specific area (rectangle) of a display.  Remote control of it isn't necessary, but this would be a defined rectangle (in code) and not a full display or a specific Window.

Is this Package capable of that?

**Comment by MarcusWichelmann** [comment, marked as answer]
Hi, this library provides a VNC Client for viewing a shared screen, not a Server for sharing one.
